modules/client-configuration/pages/clients-opensuseleapmicro.adoc

```diff
@@ -0,0 +1,123 @@
+[[clients-opensuseleapmicro]]
+= Registering {leapmicro} Clients
+
+This section contains information about registering clients running these {leapmicro} operating systems:
+
+* {leapmicro} 5.3 {x86_64}
+* {leapmicro} 5.3 {arm64}
+
+The {leapmicro} is an ultra-reliable, lightweight operating system purpose built for edge computing.
+It leverages the enterprise hardened security and compliance components of SUSE Linux Enterprise and merges them with a modern, immutable, developer-friendly OS platform.
+
+The {leapmicro} uses transactional updates.
+Transactional updates are atomic (all updates are applied only if all updates succeed) and support rollbacks.
+They do not affect a running system as no changes are activated until after the system is rebooted.
```

Comment:
```suggestion
They do not affect the running system because no changes are activated until the system is rebooted.
```

Reply:
Fine, but keep in mind this is a copy paste of what you already have for SLE Micro :-)